src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -3153,6 +3155,40 @@ paths:
         '404':
           description: |
             **nodeId** does not exist
+  '/nodes/{nodeId}/storage-info/{contentPropName}':
+    get:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Retrieve storage properties for given content
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+
+        Gets storage properties for given content.
+      operationId: getStorageProperties
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - name: contentPropName
+          in: path
+          description: The qualified property name of content (e.g. 'cm:content')
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ContentStorageInfo'
+        '400':
+          description: |
+            Invalid parameter: value of **maxItems** or **skipCount** is invalid
```

Comment:
Remove maxItems / skipCount. Looking at other content-based endpoints that return 400, maybe:

"Invalid parameter: **nodeId** is not a valid format, or is not a file"

Reply:
Thx. Changed.